### ISSUE TITLE
TD 5 

### ISSUE THREAD

**Issue by Naima-Ikhlef** [UPHF/Algo3_202223#168]
https://github.com/Naima-Ikhlef/ALGO3_2022_2023/blob/main/TD/TD%205.c

**Comment by martinezmatias** [comment]
> https://github.com/Naima-Ikhlef/ALGO3_2022_2023/blob/main/TD/TD%205.c#L127 

Merci, c'est bien, mais pas la meilleure solution: Nous avons discutter l'algo fusion ici: https://github.com/UPHF/Algo3_202223/issues/113

**Comment by Naima-Ikhlef** [comment]
Oui, mais je n'ai pas très bien compris, donc pour le moment j'ai laisser comme ça
